### PR TITLE
Fix fertilizer usage in Crop Manager

### PR DESCRIPTION
**Pull request by xSkewer**
Haven't tested ingame yet but it should work. Also increases the fertilizer cap to 200, which is what Forestry farms are at.

### CODE REVIEW

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_CropHarvestor.java [hunk 1]
```diff
@@ -421,7 +421,7 @@ public boolean applyFertilizer(ICropTile aCrop) {
 			return false;
 		} else {
 			//Logger.INFO("Current Nutrient: "+aCrop.getNutrientStorage()+" for "+aCrop.getCrop().displayName());
-			aCrop.setNutrientStorage(100);
+			aCrop.setNutrientStorage(getNutrientStorage() + 100);
```
Comment (kuba6000) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_CropHarvestor.java]: I think you forgot something
```suggestion
			aCrop.setNutrientStorage(aCrop.getNutrientStorage() + 100);
```

**Review by kuba6000 — CHANGES_REQUESTED** [review]

**Review by kuba6000 — APPROVED** [review]
Ok

**Comment by draknyte1** [comment]
> Haven't tested ingame yet but it should work. Also increases the fertilizer cap to 200, which is what Forestry farms are at.

IC2 states the max should be 100. The values for the crop manager are based on IC2. 

Forestry should be nerfed, it’s essentially giving crops free fert by setting to 200.

**Comment by xSkewer** [comment]
> IC2 states the max should be 100. The values for the crop manager are based on IC2.
> 
> Forestry should be nerfed, it’s essentially giving crops free fert by setting to 200.

You can hand apply fertilizer up to 199.

**Pull request merged**